cache/cache.go

```diff
@@ -175,7 +175,7 @@ func serveWatchEvents(ctx context.Context, watchCtx *watchCtx) {
 	backoff := watchCtx.backoffStart
 	for {
 		opts := []clientv3.OpOption{
-			clientv3.WithPrefix(),
+			clientv3.WithFromKey(),
```

Comment:
This should not be changed.

Reply:
Ok, please be clear next time to avoid confusion. No need to be so detailed. 

"Ok, will also need to update tests" is enough.